MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -9,6 +9,8 @@ protocol FeedViewControllerDelegate {
 }
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching, FeedLoadingView {
+	@IBOutlet private var errorView: ErrorView?
```

Comment:
I recommend making `errorView` getter public on the production side and referencing it instead of casting the `tableHeaderView` to `ErrorView` on the test side.

Ideally, the view controller should be decoupled from the view hierarchy in the storyboard.

This way, you can easily change the view hierarchy in the storyboard without breaking the code here.

`errorView` setter can be private so that clients of `FeedViewController` cannot change its value.

```suggestion
	@IBOutlet public private(set) var errorView: ErrorView?
```